docs/getting_started.md

```diff
@@ -280,24 +280,36 @@ brew uninstall sunshine
 
 ### Windows
 
+Sunshine now supports ARM64 on Windows; however this should be considered experimental. This version does not properly
+support GPU scheduling and any hardware acceleration.
+
 #### Installer (recommended)
 
-1. Download and install
-   [sunshine-windows-installer.exe](https://github.com/LizardByte/Sunshine/releases/latest/download/sunshine-windows-installer.exe)
+1. Download and install based on your architecture:
+
+   | Architecture          | Installer                                                                                                                                    |
+   |-----------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+   | AMD64/x64 (Intel/AMD) | [Sunshine-Windows-AMD64-installer.exe](https://github.com/LizardByte/Sunshine/releases/latest/download/Sunshine-Windows-AMD64-installer.exe) |
+   | ARM64                 | The installer is not currently supported, please use the portable version                                                                    |
```

Comment:
Please use the link as I had in my branch.

The installer will be available before this PR is merged.